McCorrections/include/LeptonWeights.h

```diff
@@ -144,7 +146,7 @@ class LeptonWeights : public IWeightProvider {
     LeptonWeights(const std::string& electron_idIsoInput, const std::string& electron_SingletriggerInput,
                   const std::string& electron_CrossTriggerInput, const std::string& muon_idIsoInput,
                   const std::string& muon_SingletriggerInput, const std::string& muon_CrossTriggerInput,
-                  const std::string& tauTriggerInput, const std::string& tauTriggerInputOld, Period period, DiscriminatorWP _tau_iso_wp,
+                  const std::string_view& tauTriggerInput, Period period, DiscriminatorWP _tau_iso_wp,
```

Comment:
use or `const std::string&` or `std::string_view`. to be coherent with other arguments I suggest to keep `const std::string&`.